### ISSUE TITLE
High level EC2 module to unblock EC2 RFC

### ISSUE THREAD

**Issue by depombo** [iasql/iasql-engine#1189]

**Comment by aguillenv** [comment]
why does the high-level EC2 block the EC2 RFC?

**Comment by depombo** [comment]
Oh I thought we were concerned that the tables would get to wide once we added all of the ec2 functionality?

**Comment by aguillenv** [comment]
yes, that's a concern in terms of UX but I wouldn't say a blocker. Also, all those properties will be added gradually, so it could also be worked in parallel.
